recipes/jupyterlab-nbconvert-nocode/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set name = "jupyterlab_nbconvert_nocode" %}
+{% set version = "0.3.1" %}
+{% set build_num = 0 %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a7c1617d2b49d425984d30e5e0cc9e7e1b354bb570dab474cafffb70f61b5162
+
+build:
+  noarch: python
+  number: {{ build_num }}
+
+requirements:
+  host:
+    - python >=3.6
+  run:
+    - python >=3.6
+
+outputs:
+  - name: {{ name|lower }}
+    build:
+      noarch: python
+      number: {{ build_num }}
+      script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+    requirements:
+      host:
+        - python >=3.6
+        - pip
+        - jupyter-packaging
+      run:
+        - python >=3.6
+        - jupyterlab >=1.0.0
+        - nbconvert >=6.0.0
+
+    test:
+      imports:
+        - jupyterlab_nbconvert_nocode
+      commands:
+        - pip check
+      requires:
+        - pip
+
+  - name: jupyterlab-nbconvert-nocode
+    build:
+      noarch: python
+      number: {{ build_num }}
+      
+    requirements:
+      host:
+        - python >=3.6
```

Comment:
Can you try removing this completely and also the python from run? This package is not doing anything so nothing will be needed. All the deps will come from the main package.

Reply:
Tried it in a prior revision, jinja error